tests/helper.py

```diff
@@ -98,3 +108,19 @@ def invoke_installed_extras_command(self, name, *params):
         script = [temp_extras_command, *params]
         print(f"Run the script \"{script}\"")
         return subprocess.run(script, capture_output=True)
+
+    def change_origin(self, origin_url):
+        try:
+            self._git_repo.git.remote("add", "origin", origin_url)
+        except GitCommandError:
+            pass
```

Comment:
We should log the error?

Reply:
The log is better but the throw is not expected.

Reply:
Update